pkg/types/installconfig.go

```diff
@@ -47,11 +47,14 @@ type InstallConfig struct {
 	// Networking defines the pod network provider in the cluster.
 	*Networking `json:"networking,omitempty"`
 
-	// Machines is the list of MachinePools that need to be installed.
+	// ControlPlane is the configuration for the machines that comprise the
+	// control plane.
 	// +optional
-	// Default on AWS and OpenStack is 3 masters and 3 workers.
-	// Default on Libvirt is 1 master and 1 worker.
-	Machines []MachinePool `json:"machines,omitempty"`
+	ControlPlane *MachinePool `json:"controlPlane,omitempty"`
```

Comment:
We'll need to bump [`InstallConfigVersion`][1] for this.  Possibly also with a temporary hack to get through CI (e.g. like the one being removed by #1251).

[1]: https://github.com/openshift/installer/blob/e1082a3ba20036be273300984ac01b891993764b/pkg/types/installconfig.go#L14

Reply:
I wasn't sure whether the version needed to be bumped with every change. Or whether the version change covered the collections of API changes between beta1 and beta2.

Reply:
We cut 0.12.0 with v1beta2, so I think we need another bump.  But I'm fine combining future changes that land between releases.

Reply:
Fair enough. I will bump up to v1beta3.